### DISCUSSION TITLE
Implications of packer license change from MPL to BSL

### DISCUSSION THREAD

**Discussion by alperkar** [actions/virtual-environments#9156] [General questions]
Packer, from version 1.9.3 is no longer provided under the MPL license. Instead it uses BSL license. What are the implications of this for the companies that use this repository to build their self-hosted build agents?

**Comment by erik-bershel** [comment]
Unfortunately, we cannot answer this kind of question. Legal issues are very shaky ground for making any kind of statements without having complete information. 
I can only recommend that you read the BSL license itself and the FAQ section on the developer’s website: https://developer.hashicorp.com/packer
